src/SignIn.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { useState } from 'react'
+import { IconButton } from "@mui/material";
+import './SignIn.css'
+import AccountCircleRoundedIcon from "@mui/icons-material/AccountCircleRounded";
+import Home from './Home';
+import { useNavigate } from 'react-router-dom'
+
+function SignIn() {
+  const history = useNavigate()
+  const [data, setData] = useState({
+    username: "",
+    password: "",
+  });
+  const inputHanlder = event => {
+    const { name, value } = event.target;
+    setData((prev) => {
+      return { ...prev, [name]: value }
+    });
+  }
+
+  // useEffect(() => {
+  //     const inputVal = JSON.parse(localStorage.getItem('formValues'));
+  //     if (inputVal) {
+  //      setInputVal(inputVal);
+  //     }
+  //   }, []);
+  const submitButton = (e) => {
```

Comment:
Form should be able to handle the form information, no need to read it from state